docker-compose-devel-volmount-default.yml

```diff
@@ -1,7 +1,10 @@
 version: '2.2'
 services:
   development:
+    privileged: true
+    pid: "host"
     volumes:
       - .:/home/centos/
       - .docker/cpanm:/home/centos/.cpanm
-
+      - /sys/kernel/debug:/sys/kernel/debug
```

Comment:
Why are these 2 needed?

Reply:
just to attach to the kernel, need to read that

Reply:
I guess this is fine, but I'm not sure I understand all the implications of this. Should they be mounted as read only?

Reply:
If it's ro, the main issue is that bpftrace cannot be installed because no space left :\  and I do not want to have bpftrace installed by default. 

Regards